src/deadline/client/ui/__init__.py

```diff
@@ -45,19 +45,22 @@ def gui_error_handler(message_title: str, parent: Any = None):
 
 
 @contextmanager
-def gui_context_for_cli():
+def gui_context_for_cli(automatically_install_dependencies: bool):
```

Comment:
Should this have a default value of False? I know the flags we pass in have a default value of False, but adding that here would assist in some safety of a new CLI being added that uses the gui but does not provide this CLI arg.